crates/turbopack-tests/tests/snapshot/css/absolute-uri-import/output/crates_turbopack-tests_tests_snapshot_css_absolute-uri-import_input_index.css

```diff
@@ -0,0 +1,14 @@
+/* chunk [workspace]/crates/turbopack-tests/tests/snapshot/css/absolute-uri-import/output/crates_turbopack-tests_tests_snapshot_css_absolute-uri-import_input_index.css */
+/* import([project]/crates/turbopack-tests/tests/snapshot/css/absolute-uri-import/input/other.css (css)) */
+
+/* [project]/crates/turbopack-tests/tests/snapshot/css/absolute-uri-import/input/other.css */
+.foo {
+  background-color: red;
+}
+
+/* [project]/crates/turbopack-tests/tests/snapshot/css/absolute-uri-import/input/index.css */
+@import "https://example.com/stylesheet1.css";
```

Comment:
This is technically invalid css, as these `@imports` are not the first rules (after `@charset`) that appear in the file. This behavior is reproducible without turbopack in non-turbo next dev.

We could hoist these declarations to the top of the chunk. I'm not sure the impact on specificity this has (or even if it would be different than the invalid css we produce today).